### ISSUE TITLE
Mac M1 tensorflow installation error

### ISSUE THREAD

**Issue by JingZhang918** [tensorflow/tensorflow#53309]
**System information**
- OS Platform and Distribution: Mac OS Big Sur Version 11.6 Apple M1 chip Memory 8G

Here is how I install:
```
conda create -n tensorflow python=3.9.5
conda activate tensorflow
conda install -c apple tensorflow-deps
conda install -c apple tensorflow-deps==2.6.0
python -m pip install tensorflow-macos
python -m pip install tensorflow-metal
brew install libjpeg
conda install -y matplotlib jupyterlab
```
Then I run:
```
python
import tensorflow as tf
```
Then I got error:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/zhangjing/miniforge3/envs/tf3.7/lib/python3.8/site-packages/tensorflow/__init__.py", line 449, in <module>
    _ll.load_library(_plugin_dir)
  File "/Users/zhangjing/miniforge3/envs/tf3.7/lib/python3.8/site-packages/tensorflow/python/framework/load_library.py", line 155, in load_library
    py_tf.TF_LoadLibrary(lib)
tensorflow.python.framework.errors_impl.NotFoundError: dlopen(/Users/zhangjing/miniforge3/envs/tf3.7/lib/python3.8/site-packages/tensorflow-plugins/libmetal_plugin.dylib, 6): Symbol not found: _OBJC_CLASS_$_MPSGraphCompilationDescriptor
  Referenced from: /Users/zhangjing/miniforge3/envs/tf3.7/lib/python3.8/site-packages/tensorflow-plugins/libmetal_plugin.dylib (which was built for Mac OS X 12.0)
  Expected in: /System/Library/Frameworks/MetalPerformanceShadersGraph.framework/Versions/A/MetalPerformanceShadersGraph
```

**Comment by Saduf2019** [comment]
@JingZhang918 
We don't support the M1 chip.
Can you refer to similar issues:#42482,46178,48808
Can you please create this on [Apple Developer Forum.](https://developer.apple.com/forums/tags/tensorflow-metal)

**Comment by jlgoh** [comment]
Can confirm that I was facing this issue on my M1 Mac with MacOS 12.0 and solved this issue after upgrading to MacOS 12.6.

Installation following this guide: https://www.mrdbourke.com/setup-apple-m1-pro-and-m1-max-for-machine-learning-and-data-science/
